automatische Rückführung

dani-0101 opened the pull request:
countDown variable can be ignored

hannah1666 committed:
adds Icon + checks if not Sat or Sun

hannah1666 committed:
adds comment: calculate the color

ich fang mal an, alles was ich im Code verstehe und brauche zu kommentieren, damit ichs nicht jedes mal wieder erneut nachvollziehen muss - hoffe das ist okay!

hannah1666 committed:
Kommentare + Versuch Farben anzugeben

Commit:
fixed bug on pass params for route

hannah1666 committed:
fixes bug

Commit:
saves temp variables in storage

hannah1666 committed:
bug fix - loading vor Arbeitsplatzauswahl eingefügt

hannah1666 committed:
Start: Ausgabe this.bookings

hannah1666 committed:
comments on various functions

hannah1666 committed:
comments & inserts Color calculation in DayPicker

thelinking committed:
Button zur Platzbuchung

- Buttondesign eingefügt
- "Platz reservieren" -> "Platz buchen"

thelinking committed:
Buttondesign angepasst

- Code kommentiert
- Abstand zwischen Text und Button eingefügt
- Farbe des Buttons
- Textgröße

hannah1666 committed:
calculates Farbanzeige Daypicker depends on capacity

hannah1666 committed:
deletes test output

hannah1666 committed:
entfernt test ausgabe agaiin

hannah1666 committed:
Update DayPicker.vue

hannah1666 committed:
important

hannah1666 committed:
important eingefügt

dani-0101 committed:
automatische Rückführung

countDown variable can be ignored

Commit:
adds error handling in api for booking

wall0404 commented on public/css/app.css:
```diff
@@ -11122,38 +11122,8 @@ a.text-dark:focus {
   margin-right: 0;
 }
 
-/* INPUT Styles */
```
Warum entfernst du das?
Das muss auf jeden Fall drin bleiben. Zum Beispiel für die Sign Up Page

dani-0101 replied:
Ich kann mich nicht erinnern irgendwas an der SCSS Datei verändert zu haben. Die Änderungen die hier angezeigt werden sich bei mir auch nicht vorhanden

wall0404 commented on public/css/app.css:
```diff
@@ -11122,38 +11122,8 @@ a.text-dark:focus {
-
 /* FORM Styles */
 
-.coba-form {
```
Das auch

hannah1666 committed:
merged develop

hannah1666 committed:
Merge branch 'CBK13-Sitzplatzbuchung-API' into CBK13-Sitzplatzbuchung

mira853 committed:
Abgelaufenen Token refreshen

thelinking committed:
Button rechtsbündig

thelinking committed:
Kommentar

thelinking committed:
Merge branch 'CBK8-Homescreen' into CBK13-Sitzplatzbuchung

hannah1666 committed:
modifies seat view so that colors are displayed and adjusts formatting

hannah1666 committed:
Merge branch 'CBK13-Sitzplatzbuchung' of https://github.com/wall0404/coba into CBK13-Sitzplatzbuchung

hannah1666 committed:
changes amount of hours when the indicator gets red & names of the color inidcators

thelinking committed:
Icon zu "Platz buchen"-Button eingefügt

hannah1666 committed:
Anpassungen Sitzplatzansicht Liste

- Namensänderung Sitzplätze
- Locationname-Angabe in Header
- Sitzplatzname unter Button
- Abstandsanpassung

hannah1666 committed:
Merge branch 'CBK13-Sitzplatzbuchung' of https://github.com/wall0404/coba into CBK13-Sitzplatzbuchung

thelinking committed:
Merge branch 'CBK13-Sitzplatzbuchung' into CBK-98

thelinking committed:
Code zur Automatischen Rückführung auskommentiert

thelinking committed:
Gründgerüst für Anzeige zur Bestätigung der Sitzplatzbuhcung

- Text mit Daumenhoch eingefügt
- 2 Button eingefügt ("Zur Übersicht", "Weiter Buchung")

hannah1666 committed:
Farbanpassung disabled day

mira853 committed:
die Navbar wurde aus Sign UP/In entfernt, dafür Buttons hinzugefügt und die es wurden Details angepasst

Commit:
adss front end slider check if already booked

Commit:
removes half an hour puffer between to bookings

mira853 committed:
Das Aussehen vom Coba button wurde angepasst

Eine Klasse wurde dafür hinzugefügt

hannah1666 committed:
Merge branch 'origin/CBK9-SignIn' into CBK13-Sitzplatzbuchung

thelinking committed:
Anpassung von Text und Icon

thelinking committed:
Anpassung der Buttons

thelinking committed:
Text angepasst

hannah1666 committed:
Info Button beiw siwtzplatzauswahl + modalöffnung mit iinfos + triwggerbox

wall0404 commented on resources/sass/coba-ci.scss:
```diff
@@ -143,9 +152,24 @@
+        background-color: #4ABE5D;
+    }
+    &.coba-button-red {
+        background-color: #FF6666 !important;
```
important raus

thelinking replied:
entfernt

wall0404 committed:
Merge pull request #38 from wall0404/CBK13-Sitzplatzbuchung

Cbk13 sitzplatzbuchung

wall0404 commented on resources/js/components/Pages/Book/Page_BookingCheckout.vue:
```diff
@@ -1,20 +1,27 @@
+                            <b-icon icon="hand-thumbs-up" font-scale="3" style="color:#FFC931"></b-icon>
+                        </div>
+                        <div class="coba-container"> <!-- Buttons -->
+                            <router-link to="/home"><button class="coba-button coba-button-white coba-button-yellow-border">Zur Startseite</button></router-link>
```
abstand einfügen
 mt-2

thelinking replied:
Abstand eingefügt

thelinking committed:
Fertigstellung

- Button abstand eingefügt
- Sitzung abgelaufen -> zur Startseite

thelinking committed:
Kommentar geändert

thelinking committed:
"!Important" entfernt

thelinking committed:
Merge branch 'CBK-98' of https://github.com/wall0404/coba into CBK-98

thelinking committed:
Sitzplatzbuchung-Button verkleinert

- coba-button-normal eingefügt

thelinking commented:
Ich muss doch noch was am Design verbessern, habs erstmal zu mir assigned

thelinking committed:
Anzeige angepasst

- Wert für padding sind jetzt prozentual, damit die Anzeige auf unterschiedlichen Geräten sind anpasst
